catalog/core/catalog-core-migratable/src/test/java/org/codice/ddf/catalog/migratable/impl/MigrationFileWriterTest.java

```diff
@@ -210,6 +180,17 @@ public void testWriteMetacardsWriteObjectFails() throws Exception {
         List<Result> results = new ArrayList<>();
         results.add(new ResultImpl(new MetacardImpl()));
 
+        mockFile = new File("");
```

Comment:
In the entire test class, File's are _never_ written out. Only stored in memory, if that. Most of the time they are mocked. It didn't make sense here since I was just trying to avoid powermock NPE's so that I could get to the logic I was testing on failure. Mocking the File just for that would have been verbose and wasteful.